common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py

```diff
@@ -0,0 +1,98 @@
+"""
+Unit Tests for Django management command to archive course community on nodeBB.
+"""
+from __future__ import unicode_literals
+
+from datetime import datetime, timedelta
+
+import mock
+import pytz
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class ArchivePastCoursesCommunities(ModuleStoreTestCase):
+    """
+    Tests for `archive_past_courses_communities` command.
+    """
+
+    def setUp(self):
+        """
+        This function is responsible for creating courses for every test and mocking the function for tests.
+        :return:
```

Comment:
We are following google style for docstrings Please update

Reply:
Fixed